### ISSUE TITLE
[HelpHub] Pull Quote

### ISSUE THREAD

**Issue by zzap** [WordPress/Documentation-Issue-Tracker#148]
Article: https://wordpress.org/support/article/pullquote-block/
## General
- [ ] make sure all screenshots are relevant for current version (5.9)
- [ ] update changelog at the end of the article

Issue migrated from Trello: https://trello.com/c/mL6H2t7u/75-pull-quote

**Comment by jaz-on** [comment]
@zzap I'll take care of this one to start with! :)

**Comment by jaz-on** [comment]
@zzap Ok I am done with the tasks.

I updated every screenshot + adding a new video tutorial.
Everything is tested on WordPress v5.9 (medias are named after the version number for consistency).
I drafted the updated changelog at the end of the post.

I found a few issues to discuss:
- first the "transform to" button has changed: there is no more options to change the styles there
- in fact, the block no longer has any style options / I think that everything related to "solid style" has to be changed
- a few editing options and toolbar options also changed: the lists must be reviewed to be sure it's up-to-date :)

Ps: I took the liberty of rewriting the name of the block in the post with only one spelling: "pullquote". All attached and without capitalization. I have no opinion on the best spelling, I just used the most used spelling. :)
